jetty-core/jetty-siwe/src/test/resources/login.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-In with Ethereum</title>
+    <script src="https://cdn.jsdelivr.net/npm/web3@1.6.1/dist/web3.min.js"></script>
+</head>
+<body>
+<h4>Sign-In with Ethereum</h4>
+<button id="siwe">Sign-In with Ethereum</button>
+<form id="loginForm" action="/auth/login" method="POST" style="display: none;">
+    <input type="hidden" id="signatureField" name="signature">
+    <input type="hidden" id="messageField" name="message">
+</form>
+<p class="alert" style="display: none;">Result: <span id="siweResult"></span></p>
+
+<script>
+    let provider = window.ethereum;
+    let accounts;
+
+    if (!provider) {
+        document.getElementById('siweResult').innerText = 'MetaMask is not installed. Please install MetaMask to use this feature.';
```

Comment:
So this test needs MetaMask installed and a MetaMask account installed? If it can't run as part of the normal build/test cycle, it will rot over time (ask me how I know).

Reply:
This is only for `SignInWithEthereumEmbeddedExample` which is not run as a test but a demo/example.

No way to test this with a unit test as it requires browser interaction. But it is good to have this code around somewhere so we can actually try out this module with real requests from the browser.